packages/appstore/src/components/override-other-packages-styles-to-make-the-code-even-more-unclear-because-everything-is-asap.scss

```diff
@@ -0,0 +1,21 @@
+.traders-hub {
```

Comment:
we should just go and fix in trades-hub style directly instead of overriding.

Reply:
@henry-deriv are you sure? Do you want I change common `Button` component from `components` package which used everywhere? The same with `ThemedScrollbars` component? I don't think we can break everything... We are merging our changes step by step, not all together, so I can't change common components to prevent breaking changes in other packages

Reply:
I know it's frustrating we're being rushed, but we can't commit this filename. If we really do need to override the styles, we can call it `temporary-overrides.scss`, and then create a tech debt card, or add a subtask to the card that fixes it.

Reply:
@markwylde-deriv Sure :) I like your suggested name, I'll use it)